packages/go_router/pubspec.yaml

```diff
@@ -7,7 +7,7 @@ issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+
 
 environment:
   sdk: ">=2.17.0 <3.0.0"
-  flutter: ">=3.0.0"
+  flutter: ">=3.1.0-0.0.pre.2106"
```

Comment:
Is this intended? Does this mean folks will need to use the beta/master release channel until we release the next version of Flutter?

Reply:
yes this is just a placeholder to make ci happy. go_router_v5 feature branch will merge into main after next stable release